### PR TITLE
Style/details

### PR DESCRIPTION
**Pull request by Thee-Dust**
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant screenshots of your code being deployed. 
- adds style to details page and some media queries 

Fixes # (issue)
- #

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/components/PokemonDetails/PokemonDetails.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const PokemonDetails = ({id, caught, favorite}) => {
           </div>
           <div className='pokemon-container'>
             <div className="pokemon-holder">
-              <img src={`https://pokeres.bastionbot.org/images/pokemon/${id}.png`} alt={pokemonDetails.name} className='pokemon' />
+              <img src={`https://play.pokemonshowdown.com/sprites/xyani/${pokemonDetails.name}.gif`} alt={pokemonDetails.name} className='pokemon' />
```
Comment (rhen92) [inline src/components/PokemonDetails/PokemonDetails.js:48]: I like the use of the gif in the details page!

**Comment by rhen92** [comment]
Looks great Dustin!!

File: src/components/Home/Home.css [hunk 2]
```diff
@@ -1,12 +1,14 @@
 .card-display {
   background-color: white;
   width: 95%;
-  display: grid;
-  grid-template-columns: repeat(5, 1fr);
-  grid-gap: 20px;
-  text-align: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
```
Comment (asiisii) [inline src/components/Home/Home.css:6]: nice use of flex here to center cards at the bottom row

File: src/components/Navbar/Navbar.css [hunk 3]
```diff
@@ -26,4 +26,23 @@ a {
...
+  color: white;
+}
+
+.log-out {
+  border: none;
+  background: none;
+  font-size: 36px;
+  font-weight: bold;
+  color: #cdcdcd;
+  cursor: pointer;
```
Comment (asiisii) [inline src/components/Navbar/Navbar.css:41]: good job thinking about UX here

File: src/components/PokemonCard/PokemonCard.css [hunk 4]
```diff
@@ -27,13 +29,32 @@
...
+
+@media screen and (max-width: 1600px) and (min-width: 1200px) {
+  .pokemon-card {
+    width: 24%;
+  }
+}
+
+@media screen and (max-width:1200px) and (min-width: 800px) {
+  .pokemon-card {
+    width: 32%;
```
Comment (asiisii) [inline src/components/PokemonCard/PokemonCard.css:57]: ah that's how we can make our app responsive. Good job

**Review by asiisii — APPROVED** [review]

**Pull request merged**